taskcluster/translations_taskgraph/actions/train.py

```diff
@@ -343,6 +330,32 @@ def train_action(parameters, graph_config, input, task_group_id, task_id):
 
     parameters = dict(parameters)
 
+    start_stage = input.pop("start-stage", None)
+    if start_stage:
+        previous_group_ids = input.get("previous_group_ids")
+
```

Comment:
What happens if `previous_group_ids` is `None`? Should we handle that case and throw a `ValueError` (or let it throw a `KeyError`)?

Reply:
+1. I'll throw an error for this case. _Technically_ we could set it to an empty list and it would work...but it wouldn't do what we expected.